content/es/canary_deployment.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Canary Deployment
+status: Completed
+category: Concepto
+---
+
+## ¿Qué es?
+
+Canary deployment es una estrategia de despliegue que comienza con dos entornos: uno con tráfico en vivo y otro que
```

Comment:
Tráfico en vivo se entiende? En inglés si que se suele usar el tema de `going live`, pero en castellano no tanto. Normalmente se suele usar la expresión con tráfico / tráfico real y sin tráfico. Qué opináis?

cc @anyulled @electrocucaracha @krol3

Reply:
Creo que eliminando la parte de `en vivo` se sigue entendiendo el concepto.

`uno con tráfico y otro que contiene el código actualizado sin tráfico.`